playwright/tests/desk-tool/documentActions.spec.ts

```diff
@@ -0,0 +1,113 @@
+import {test, expect} from '@playwright/test'
+import {createUniqueDocument, DEFAULT_DATASET, deleteDocument} from '../../helpers'
+
+/* 
+  **NOTE**
+  In these tests, we are not testing input events, so to speed up the tests we will use use `page.fill()` instead of `page.type()` to populate the document title field 
+*/
+
+const DOC_TITLE = 'Document actions [Playwright]'
+const inputSelector = 'data-testid=input-title >> input'
+
+// The path of the network request we want to wait for
+const documentActionPath = (action) =>
+  `*/**/mutate/${DEFAULT_DATASET}?tag=sanity.studio.document.${action}*`
+
+// Helper to generate path to document created in a test
+const generateDocumentUri = (docId) =>
+  `/${DEFAULT_DATASET}/desk/input-ci;documentActionsCI;${docId}`
```

Comment:
The first part here isn't actually the dataset name, but rather the name of the space - maybe we can define a separate constant for it?

```suggestion
const generateDocumentUri = (docId) =>
  `/${STUDIO_SPACE_NAME}/desk/input-ci;documentActionsCI;${docId}`
```